decidim-admin/app/views/layouts/decidim/admin/_sidebar_menu.html.erb

```diff
@@ -2,7 +2,16 @@
 <div class="sidebar-menu__container">
   <div class="sidebar-menu">
     <div id="admin-sidebar-menu-settings">
-      <%= sidebar_menu(secondary_root_menu).render %>
+      <div class="hidden md:block">
+        <%= sidebar_menu(secondary_root_menu).render %>
+      </div>
+        <button class="button button__sm button__secondary w-16 mb-3" data-component="dropdown" data-target="dropdown-menu-sidebar">
+          <%= t("menu", scope: "decidim.admin.titles") %>
+        </button>
+
+        <ul id="dropdown-menu-sidebar">
+          <%= sidebar_menu(secondary_root_menu).render %>
+        </ul>
```

Comment:
Try it out locally and let me know what you think: 

```suggestion

      <button data-component="dropdown" data-target="dropdown-menu-sidebar" data-auto-close="true" data-disabled-md="true" data-scroll-to-menu="true">
        <span><%= t("menu", scope: "decidim.admin.titles") %></span>
        <%= icon "arrow-down-s-line" %>
        <%= icon "arrow-up-s-line" %>
      </button>
      <ul id="dropdown-menu-sidebar" class="md:hidden">
        <%= sidebar_menu(secondary_root_menu).render %>
      </ul>
```

Reply:
@andreslucena will update and push this evening.